app.py

```diff
@@ -183,6 +186,22 @@ def __init__(self):
         print("")
         print(banner)
 
+    def load_product_file(self, fname):
+        """
+        loads the product file into a dictionary called product_list where the key is productID and the value 
+        is the pair (X, Y).
+        """
+
+        f = open(fname, 'r')
+        next(f)
+        
+        for line in f:
+            fields = line.strip().split()
+            self.product_info[fields[0]] = fields[1] , fields[2]

```

Comment:
Cast these to ints, for alpha, we 're not going to care about decimal values. We'll prob change this in the future.
```suggestion
            self.product_info[int(fields[0])] = (int(float(fields[1]) , int(float(fields[2]))
```